pkg/resource/transform_job/testdata/v1alpha1/readone/observed/late_initialize.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: TransformJob
+metadata:
+  creationTimestamp: null
+  name: unit-testing-transform-job
+spec:
+  dataProcessing:
+    inputFilter: $
```

Comment:
$?

Can we remove this $ and null fields?

Reply:
the $ comes from the custom_delta.go file for this resource

Reply:
looks like something related to response from service https://github.com/aws-controllers-k8s/sagemaker-controller/blob/main/pkg/resource/transform_job/testdata/sdkapi/describe/inprogress_describe.json#L6-L8